lib/api/core/entrypoints/embedded/index.js

```diff
@@ -97,24 +97,15 @@ class EmbeddedEntryPoint extends EntryPoint {
     this.httpServer = http.createServer();
     this.httpServer.listen(this.config.port, this.config.host);
 
-    if (this.config.protocols.http.enabled) {
-      this.protocols.http = new HttpProtocol();
-      this.protocols.http.init(this);
-    }
-
-    if (this.config.protocols.mqtt.enabled) {
-      this.protocols.mqtt = new MqttProtocol();
-      this.protocols.mqtt.init(this);
-    }
-
-    if (this.config.protocols.websocket.enabled) {
-      this.protocols.websocket = new WebSocketProtocol();
-      this.protocols.websocket.init(this);
-    }
-
-    if (this.config.protocols.socketio.enabled) {
-      this.protocols.socketio = new SocketIoProtocol();
-      this.protocols.socketio.init(this);
+    this.protocols = {
```

Comment:
Unrelated change: the test on the `enabled` boolean property made in the previous version is duplicated in all protocols implementations, so I decided that protocols were responsible of their own configuration and deduplicated the test by removing the ones made by the global entrypoint object.